### DISCUSSION TITLE
Comparing 3D-files

### DISCUSSION THREAD

**Discussion by haudankaivajasi** [qarmin/czkawka#655] [Ideas]
Hi. I didn't find any mention about comparing 3D-files, but I'd love this to have an option to be able to compare .stl-files and show a thumbnail of it to quickly assess if it is the same model without opening it.

**Comment by hrxn** [comment]
How do you expect this program to render content from a 3D file format?

**Reply by haudankaivajasi** [reply]
> How do you expect this program to render content from a 3D file format?

By implementing an stl thumbnail viewer: something like [this](https://github.com/unlimitedbacon/stl-thumb)?

Edit: or would it be easier to to just first generate the pictures/thumbnails and then compare them?
